README.md

```diff
@@ -22,6 +22,7 @@ To update to a new version, update
 - update the CSV to be downloaded into `primap-hist` in the Makefile,
 - update the filename and Zenodo record in `primap-hist/scripts/constants.py`
 - update changelog in `primap-hist/template.jinja2.html`
+- copy the replace the changelog files in `docs/changelog/` and rename the html file in the `html` directory to `index.html`.
```

Comment:
Copy or replace? Something seems duplicated.

Reply:
thanks, fixed it